### PR TITLE
patch4: add options page with svelte

### PR DESCRIPTION
**Pull request by vanvuongngo**
- options page with svelte
---
image
---
image

### CODE REVIEW

File: src/options/options.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Options from './Options.svelte';
```
Comment (jatinhemnani01) [inline src/options/options.js:1]: it should be _options.svelte_ instead of _Options.svelte_

**Review by jatinhemnani01 — APPROVED** [review]

File: src/options/options.js (hunk 1 above)
Comment (vanvuongngo) [reply src/options/options.js:1]: Hi @jatinhemnani01,
thanks for your awesome project. Do you have a roadmap?
It is strange that the filename is in lower case. This line is the same like './App.svelte'. I will try to camelcase the filename.

Comment (jatinhemnani01) [reply src/options/options.js:1]: it's ok, after done coding you can _npm run build_ to check if everything is working fine
Thanks for your contribution.

**Review by jatinhemnani01 — COMMENTED** [review]
css and options.svelte error

**Comment by vanvuongngo** [comment]
yes, because there are no styles, I will add a dummy css

**Comment by jatinhemnani01** [comment]
> yes, because there are no styles, I will add a dummy css

great. Thanks

**Pull request merged**